src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -26,6 +26,21 @@ NNVM_REGISTER_OP(_backward_copy)
   })
 .set_attr<FCompute>("FCompute<cpu>", IdentityCompute<cpu>);
 
+MXNET_OPERATOR_REGISTER_UNARY(BlockGrad)
+.MXNET_DESCRIBE("Get output from a symbol and pass 0 gradient back")
+.set_attr<FCompute>("FCompute<cpu>", IdentityCompute<cpu>)
+.set_attr<nnvm::FGradient>("FGradient",
+  [](const nnvm::NodePtr& n, const std::vector<nnvm::NodeEntry>& ograds) {
+    return MakeGradNode("_backward_BlockGrad", n, {}, {});
```

Comment:
I think what @tqchen meant was that you can use _zeros here so that the graph optimizer can do more optimization. Also you don't need to write ZeroGradCompute anymore